README.md

```diff
@@ -16,9 +16,9 @@
 [Contributing](https://github.com/pykale/pykale/blob/master/CONTRIBUTING.md) |
 [Discussions](https://github.com/pykale/pykale/discussions)
 
-PyKale is a machine learning library that leverages knowledge from multiple sources for accurate and *interpretable* prediction. It supports graphs, images, and videos now. It is based on [PyTorch](https://pytorch.org/) and several other libraries but differs from existing ones by adopting a unified *pipeline-based API* design, enforcing *standardization* and *minimalism*, and incorporating key recent developments. See the [Trello board](https://trello.com/b/X8VBNAvf/pykale-api-overview) for an overview.
+PyKale is a [PyTorch](https://pytorch.org/) library that offers reusable modules for general machine learning on graphs, images, and videos. By adopting a unified *modular API* design similar to that of [torch.nn](https://pytorch.org/docs/stable/nn.html), PyKale enforces *standardization* and *minimalism*. The main goal of PyKale is to incorporate key recent developments. See the [Trello board](https://trello.com/b/X8VBNAvf/pykale-api-overview) for an overview.
```

Comment:
1. Indeed, we are more pytorch now, even MPCA has a pytorch version in development.
2. General ML is still too broad and we need to put [specific ML topics](https://neurips.cc/Conferences/2020/PaperInformation/SubjectAreas) here. I will change it to multimodal and transfer learning.
3. Pipeline-based is beyond just modular. Our API is organized following a typical ML pipeline/workflow, different from other modular designs.
4. I think the `torch.nn` module is more complex than our `kale` module, which has an _order_ following the pipeline/workflow. It actually resembles the PL standard workflow more, but the PL APIs are not organized following the workflow.
5. Reusable modules: I think all libraries are meant to be reusable.
6. The main goal is the start of the (previous) next paragraph (now merged with Par 1). Incorporating recent development is quite generic (many libraries are doing).